src/convertToObject.js

```diff
@@ -6,7 +6,14 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .filter((style) => style.trim() !== '')
```

Comment:
Before adding a key with a property in ur reduce method, u can check if `ur properties are !== undefined`. If not, add a key. Then u won't need a new search
```suggestion
```